integration/testutil.go

```diff
@@ -721,8 +803,12 @@ func teardown(cleanups []func() error) error {
 // middleSizeLayerInfo finds a layer not the smallest or largest (if possible), returns index, size, and layer count
 // It requires containerd to be running
 func middleSizeLayerInfo(t *testing.T, sh *shell.Shell, image imageInfo) (int, int64, int) {
-	sh.O("nerdctl", "pull", "-q", "--platform", platforms.Format(image.platform), image.ref)
+	if image.mirror {
+		sh.O("nerdctl", "pull", "-q", "--platform", platforms.Format(image.platform), image.ref)
+	} else {
+		sh.O(append([]string{"ctr", "i", "import", "--platform", platforms.Format(image.platform)}, fmt.Sprintf("testImages/%s.tar", image.imgName))...)
```

Comment:
why does only these 3 places have this `if else` logic (e.g. why not other `nerdctl pull` usage)